pkg/app/master/commands/common.go

```diff
@@ -260,6 +261,33 @@ func RunHostExecProbes(printState bool, xc *app.ExecutionContext, hostExecProbes
 	}
 }
 
+func LoadParamsFromFile(c *cli.Context) error {
+	filePath := c.String(FlagCommandParamsFile)
+	if filePath == "" {
+		return nil
+	}
+
+	fileContent, err := os.ReadFile(filePath)
+	if err != nil {
+		log.Errorf("LoadParamsFromFile(%s): error reading file: %v", filePath, err)
+		return err
+	}
+
+	var params map[string]interface{}
+	err = json.Unmarshal(fileContent, &params)
+	if err != nil {
+		log.Errorf("LoadParamsFromFile(%s): error parsing file: %v", filePath, err)
+		return err
+	}
+
+	for key, value := range params {
+		log.Debug("LoadParamsFromFile(): setting key=", key, " value=", value)
+		c.Set(key, value.(string))
```

Comment:
I am not yet sure about the behavior of this line with other types. I still need to do some testing.

Reply:
![image](https://github.com/slimtoolkit/slim/assets/4891695/fa2f6a5a-c2c8-4f9b-a2c0-9603246acc73)

Here's a translation of your text to English:

"Here is an example of the behavior of the feature with values within a file in addition to those given with the command. We see that we are obliged to provide the values within a String and not an array. Is this behavior sufficient to meet the feature or does it need to be aligned?"